From c76410275933ff05701cda346944e30498d27b2e Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Wed, 09 Jun 2010 17:12:42 +0000
Subject: Never unload modules once init() has been done

Because they might register new types, and when they're unloaded,
kaboom!
---
diff --git a/src/nautilus-sendto-command.c b/src/nautilus-sendto-command.c
index dd21620..92b96bb 100644
--- a/src/nautilus-sendto-command.c
+++ b/src/nautilus-sendto-command.c
@@ -665,8 +665,6 @@ nautilus_sendto_plugin_dir_process (const char *plugindir)
 				if (p->info->init(p)) {
 					plugin_list = g_list_append (plugin_list, p);
 				} else {
-					if (!p->info->never_unload)
-						g_module_close (p->module);
 					g_free (p);
 				}
 			}
--
cgit v0.8.3.1
